examples/guestbook/README.md

```diff
@@ -620,21 +649,6 @@ For Google Compute Engine details about limiting traffic to specific sources, se
 [cloud-console]: https://console.developer.google.com
 [gce-firewall-docs]: https://cloud.google.com/compute/docs/networking#firewalls
 
-### Step Seven: Cleanup
-
-If you are in a live kubernetes cluster, you can just kill the pods by stopping the replication controllers and deleting the services.  Using labels to select the resources to stop or delete is an easy way to do this in one command.
-
-```console
-kubectl stop rc -l "name in (redis-master, redis-slave, frontend)"
-kubectl delete service -l "name in (redis-master, redis-slave, frontend)"
-```
-
-To completely tear down a Kubernetes cluster, if you ran this from source, you can use:
-
-```console
-$ <kubernetes>/cluster/kube-down.sh
-```
-
 ### Troubleshooting
```

Comment:
Refer to:

https://github.com/kubernetes/kubernetes/blob/master/docs/troubleshooting.md